migration/src/m20230708_151756_rpc_accounting_free_usage_credits.rs

```diff
@@ -0,0 +1,40 @@
+use sea_orm_migration::prelude::*;
+
+#[derive(DeriveMigrationName)]
+pub struct Migration;
+
+#[async_trait::async_trait]
+impl MigrationTrait for Migration {
+    async fn up(&self, manager: &SchemaManager) -> Result<(), DbErr> {
+        manager
+            .alter_table(
+                Table::alter()
+                    .table(RpcAccountingV2::Table)
+                    .add_column(
+                        ColumnDef::new(RpcAccountingV2::SumInclFreeCreditsUsed)
+                            .decimal_len(20, 10)
+                            .not_null(),
```

Comment:
should this have `.default(0)`?

Reply:
another inconsistency i found in the table; sometimes its marked as .default("0.0"), .default(0.0), and one time not at all